zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/AdHocSubprocessActivityController.java

```diff
@@ -50,6 +54,22 @@ public ResponseEntity<AdHocSubprocessActivitySearchQueryResult> findActivatableA
     }
   }
 
+  @CamundaPostMapping(path = "/{adHocSubprocessInstanceKey}/activate")
+  public CompletableFuture<ResponseEntity<Object>> activateActivities(
+      @PathVariable("adHocSubprocessInstanceKey") final Long adHocSubprocessInstanceKey,
+      @RequestBody final AdHocSubprocessActivateActivitiesInstruction request) {
```

Comment:
TODO add some validation on the parameters (e.g. list of flow node IDs should not be empty).